civiwiki/frontend/static/views/groups_list_view.js

```diff
@@ -0,0 +1,52 @@
+var GroupsListView = Backbone.View.extend({
+
+    el: '#groups',
+
+    initialize: function (options) {
+        options = options || {};
+        this.userModel = options.userModel;
+        this.render();
+    },
+
+    render: function () {
+        // Sample Group Data
+        this.renderSampleGroups();
+
+        // To Render User Group Data
+        //_this.renderUserGroups();
+    },
+
+    events: {
+    },
+
+    renderSampleGroups: function(){
+        // Sample Data
+        var sample_data = JSON.stringify([
```

Comment:
Hahaha why two lines in a row do you JSON stringify and then parse?  I don't think you need to do that at all